README.md

```diff
@@ -113,31 +113,24 @@ activation is possible.
 Compiling ratbagd
 -----------------
 
-ratbagd needs systemd >= 227. If you are running Fedora 23, you might find the
-following instruction valuable:
+ratbagd uses the meson build system, see http://mesonbuild.com which in
```

Comment:
The comma looks misplaced. I'd use:

"ratbagd uses the meson build system (see http://...), which in..."

...but maybe that's just my German grammar showing through.